rca/static_src/javascript/programmes/components/ProgrammesResults.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { programmePageShape } from '../programmes.types';
+
+import ProgrammeTeaser from './ProgrammeTeaser';
+
+const ProgrammesResults = ({ programmes, hasActiveSearch }) => {
```

Comment:
This is a placeholder just for the form to be tested meaningfully – not worth reviewing now.